### ISSUE TITLE
🚨 Looking for maintainer(s)!

### ISSUE THREAD

**Issue by kalkih** [kalkih/mini-graph-card#527]
As many of you probably have noticed, this project hasn't been very active in the past ~ 6 months.

I currently do not have enough time on my hands to give it the proper attention that I feel like this project deserves.
I'm therefore looking for one or more people that would be interested in **helping** / **maintaining** / **reviewing** / **developing** this project.

Contact me its@kvrl.me if you are interested.

Altough I'm less active nowadays I'll continue to make sure the card doesn't break with new releases of Home Assistant and maintain as time allows.

Thank you to all users & contributors!

**Comment by RomRider** [comment]
I'd be happy to support, but I don't have your level in JS :)

**Comment by kalkih** [comment]
Great!
Invited you as a collaborator, feel free to @ me, email or contact me on discord at any time Karl#9031 if you want to ask or discuss anything.

**Comment by OmgImAlexis** [comment]
I'd be happy to help with keeping the automated PRs merged and general housekeeping.

**Comment by jlsjonas** [comment]
I'd be happy to help maintain the project (as it's my preferred graph card); also setting up some stale automations to keep the open issues & PRs relevant.

**Comment by ildar170975** [comment]
I am trying to help with resolving issues on GitHub and in the Community thread.

**Comment by jlsjonas** [comment]
Assigning to avoid this ticket going stale again

**Comment by Liquidmasl** [comment]
I would like to help as well, although I am also full time working and maintaining some other things, so wont be super active, but help when i can.

**Comment by ildar170975** [comment]
@Liquidmasl 
Thanks for your desire to contribute!)
Would be great if you test and review some of recent PRs.

**Comment by Liquidmasl** [comment]
> [@Liquidmasl](https://github.com/Liquidmasl) Thanks for your desire to contribute!) Would be great if you test and review some of recent PRs.

I am on it, but I am a bit puzzled about the state of the repo, a lot of PRs are absolutely tiny, easy to review and test, yet in the last year there have only really been 3 merges to main. 
What is the actual breaking factor? 
I feel like a lot of PRs can be closed as douplicates, some can be merged, some can be merged with minimal changes. 

I do like this card, so I would be willing to do that. 
I could also offer to fork this completely if that helps with breaking factors..

**Comment by ildar170975** [comment]
> I could also offer to fork this completely if that helps with breaking factors..

You can fork the card by yourself. We are (a few maintainers here) not after making a fork, we are trying to maintain THIS repo. 

> What is the actual breaking factor?

Perhaps personal lives)))

**Comment by Liquidmasl** [comment]
Yeah you misunderstood my questions. I totally understand that priorities often come in the way, thats fine. 
But as I said, there are a lot of open PRs that are duplicates, and even more that are very very simple that fix issues that persist for multiple years. And there is activity from the maintainers but nearly zero merges. So something aint working out! 
I feel like real issues are not fixed cause of minor cosmetic smells. Leading to a standstill. 
Is it because to little maintainers have merge rights? Or rights to close issues and PRs? 
I even saw PRs that just include fixes from 5 other PRs, that are just stale. Which leads to even more delays and standstills. In practice this repo is choking on backlog.

The fork was an offer, if this can help getting things moving. I would prefer not to fork. But if single line code changes that fix issues that persist for mutliple years, are not merged, then it might be necessary?
I just feel like a lot of real bugs that are in there for years can be fixed in an afternoon, with actual reviewing and active manual testing. Its a tiny codebase!

**Comment by Liquidmasl** [comment]
Okay I guess I give up.
I understand now that the issue this repo has is not based in code, its based in entitlement of the maintainers. 
And also kinda with the expectations they have for themselves in their role maybe?

To engage with a few points @ildar170975 mentioned (https://github.com/kalkih/mini-graph-card/pull/1155#issuecomment-4263973347), and I am sorry if I lost a little patience in the meanwhile but...

1. "Named a PR messy and refused to explain why":
you are referring to this: https://github.com/kalkih/mini-graph-card/pull/1322#issuecomment-4227584652
Everything was already discussed by you and other people in the thread. This was a Bump with an additional 'vote' so to say. 
Thats why I proposed "I would close this in favor of https://github.com/kalkih/mini-graph-card/pull/1308" 
> some context... This is is a 6 months old PR, that is a duplicate of 2 other PRs (both of which i reviewed more thoroughly), that is just implemented worse then the others and as @ildar170975 rightfully already said in this [comment](https://github.com/kalkih/mini-graph-card/pull/1322#issuecomment-3644934945) the PR is redundant. 
I could spend more time on it, but this is, in my opinion, wasted. And I think a huge problem in this repo is that A LOT of time and energy is wasted on duplicates. So This was my 2c. And I stand by it, So in support of your argument: close the PR and invest energy into the one with more Merritt. Priorities!

 2. "Left, let us say, “unclear” comment to another PR": 
This referrs to this https://github.com/kalkih/mini-graph-card/pull/1233#issuecomment-4227529036. 
I concede that this comment does not contribute to the PR. But I wanted to display an issue with the repo. If PRs start to bunch of other unmerged PRs, this is a very very bad sign for the state of the repository it signifies a choking up of the pipeline. It makes reviwing harder and also less transparent. It should not happen. PRs should be minimal and self contained. 
This is **especially** worrisome if the PR is from the maintainer, from a repository that has not released a version in 11 **Months.**
<img width="488" height="102" alt="Image" src="https://github.com/user-attachments/assets/ba527686-f1e6-4558-b9ec-1d54183febbb" />


3. "Started claiming without concrete arguments and in demanding form that one PR must be closed, one one - merged.":
Thats just not true. I explained my reasoning multiple times. Those reasons do not deserve a long explanation tho, it it was always the same. Here again: **The PRs are duplicities, close them in favor of one of them.**
This is especially easy because the PR where i said that are all extremely tiny. 

> Sorry, no. You are not contributing. 

I am, or at least I tried, you are just not maintaining, you are entitled and in need for control, you dont let me help. I gave simple and short advice on the PRs that are simple and short, and dont need more time or energy. And I posted more in depth reviews on other PRs.
To be more precise, I reviewd the PR that should be reviewed, and I suggested closing the duplicates of it. Why? because its more efficient. It does not make sense spending time and energy on duplicates. 
https://github.com/kalkih/mini-graph-card/pull/1139 (which should also be close in favor of #1199
https://github.com/kalkih/mini-graph-card/pull/1199

> Update: repeat, nobody blocking things here, we are discussing all stuff with akloeckner - person whose PR you liked more; so please do not panic, just wait a bit.
Once again - please be patient.

The PR you commented this on was opened 2024.
The card doesnt play nice with Sections for at least this long. 
There is a timeframe where patience makes sense, and it is proportional to the projects complexity and stake. 
This project has a tiny scope, and very very small stake.


>> since this is open source, and this is very close to being unmaintained, maybe it just have to be forked.

> If this is a threat - go ahead. Create your own fork. Or just wait. Like I am currently waiting for approval of PRs from another maintainer.

OMG how much mental gymnastics does it need to not get the point. 
No its not a threat. I gain NOTHING from owning a fork. But I think this is a **great card**. And it is neglected and barely maintained. And its making issue in my dashboards. I have the skills and motivation to fix those issues, but you just wont let me help in any way. In the opposite, you defend and discourage any contribution. So no I dont want to "hostile takeover" the project. I just want the card to work for me, and all the other people who still suggest this card, even though its broken and neglected.  


And when I try to add my 2c in general, @ildar170975 is just stubborn and wants it his own way https://github.com/kalkih/mini-graph-card/pull/1128#issuecomment-4227267851. Like not implement UI stuff cause.. reasons. Even though, when spending time in the HA discord or their github, it is quite clear that HA tries to go towards UI as much as possible. 

One last time
# The issues I see in this Repo
- There has not been a merge to main in **4 months**
- There has not been a released version in **11 months**
- There are **132 Issues**. some of them from 2019, **7 years old**
- There are **28 PRs**, a lot of them duplicates, some of them with simple fixes that fix issues that persist for **2 years**.
- This card does not work correctly with section view, which is the default in HA for a long time now.
- The maintainer I interacted most with is reluctant to accept help if its not blind approval.
- The maintainer I interacted most with just _knows better_ 
- The strive for excellence is standing in the way of actually getting shit done. (Simple issues and fixes are dissected for multiple years instead of actually fixing an issue. )


And @ildar170975, comments like `Please be patient, not all of us have a luxury to sit on a cozy sofa and develop a favorite card` are just plainly uncalled for. You clearly have more time on your hands then you let on. In march you opend 8 issues, 8 PRs and 8 commits in a few repos (nice 8 consistency here) in march alone. And still the Repo you maintain is stale for months, soon a year, while you talk down on people trying to help.

**Comment by ildar170975** [comment]
@Liquidmasl 
Your posts were too demanding. You seem to not hear what I told you several times - “please wait a bit”. Please cool down. And do not measure a spare time of people which you do not know, as well as their priorities, as well as where they live and what they do.
If you want to contribute - take an open bug issue and create a PR, or report an issue. It is more productive than pushing on people here “close this, merge this”.

**Comment by Liquidmasl** [comment]
they were not demanding they were suggesting. They are trying to take weight off your shoulders not put more onto them. They are intentionally concise and on the point to make your work easier and faster, not put more onto your plate.
I tried to assist.
And the "close that" comments were just on the 2 duplicates. That should, indeed, be closed.

And no; creating new PRs while simple bugs are untouched for 2 years, and no version was released in a year is definitely not going to help.
Creating issues to throw on the issue pile will also not help.

Again. There are bugs that exist for 2+ years. With multiple simple PRs that fix them. And there hasn't been any merge in 6 Months.

> And do not measure a spare time of people which you do not know, as well as their priorities, as well as where they live and what they do.

You are joking with that comment here.... The "cozy sofa" comment came from you. I never once assumed or taken any of that into account. And AGAIN. I don't expect you to work more or faster or anything. Just not be prepotent and accept help and advice. And if you do not have the capacity to maintain then especially accept help instead of blocking an awesome repository.
In the same time you refused help and wrote comments we could have merged, tested, delivered quite a few patches. 
And I don't care where you live and what you do in the context of this repo. I care about the repo and advocate for it. If life stands in the way - okay, thats fine, then let me help. Listen to advice. Give up responsibility.

I am very sorry but I think you are either clueless, stubborn or both. There is no point for anyone to contribute here if nothing is ever accepted.

And I have no problem with patience, I contributed to HA-Core, PRs take ages over there. But maintainers there are qualified, respectful and efficient.

Good luck with the repo, and I hope you see whats going wrong at some point. I guess experience will help.

I honestly, and I am sorry, think you have very little qualification for the role or very little experience.

bottom line:
There is no justification for 3 parallel PRs none even 100 lines changed, tackling the same 2 year old issue, parallel to each other. None. No matter if you like my input or not. 
If you don't have the time/capacity or to much else is going on in life (fair enough); pass the baton, or accept help.

I am sorry that I cared, and didnt't just walk away, like most people who tried and failed to contribute here.

**Comment by ildar170975** [comment]
@Liquidmasl 
Here is what you did:
1. The only PRs where you left meaningful comments are https://github.com/kalkih/mini-graph-card/pull/1199, https://github.com/kalkih/mini-graph-card/pull/1139.
2. In https://github.com/kalkih/mini-graph-card/pull/1155 you only demanded "close it" (although I explained you why it was created & why it is still kept).
3. In https://github.com/kalkih/mini-graph-card/pull/1322 you only called this PR as "messy" with no explanations.

Sorry, no time to read long blaming essays with insults etc.
If you still want to contribute - you are always welcome with constructive reviewing of other PRs, proposing new PRs, reporting issues.
Next blaming essay will be hidden, sorry.

**Comment by kattcrazy** [comment]
Is this card open to AI coded but human managed contributions? My coding skills are minimal but this is a pretty awesome card for my dashboard and I'd love for some features to be added such as labels/lines and grid_layout support. Alternatively, I'm happy to test/merge PRs if thats helpful? However I have no interest in being caught up in drama.
